components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -813,5 +819,27 @@ private String getCallbackUrlFromInitialRequestParamMap(AuthenticationContext co
 
         return null;
     }
+    private static String getMultiAttributeSeparator() {
+
+        String multiAttributeSeparator = null;
```

Comment:
Shall we remove this null assignment and possible (not with current logic) null return of the function. Suggestion - Check whether **_MULTI_ATTRIBUTE_SEPARATOR_** is not blank and if so simply return it. Else, return **_MULTI_ATTRIBUTE_SEPARATOR_DEFAULT_**  value. This should avoid _**multiAttributeSeparator**_ usage and possible null return

Reply:
Fixed